optimizer/spec/transformers/AddBlurryImagePlaceholders/adds_blurs_on_image_passing_hueristic/expected_output.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
```

Comment:
I think you can just augment the existing test to verify all of the test cases. Also, "heuristic" is misspelled in the folder name.

Reply:
Sorry for the confusion, but are you asking me to add of the heuristic checks into the adds_blurred_placeholder test, and keep the other tests as they are, or are you asking me to consolidate all 5 tests into a single test or something else?

Reply:
Just thought this could be in fewer files -- it makes reading easier. Not a huge deal though.